modules/4337/CHANGELOG.md

```diff
@@ -14,20 +14,35 @@ Solidity optimizer: enabled with 10.000.000 runs
 
 The official deployments support the EntryPoint v0.7.0 with the canonical deployment at `0x0000000071727De22E5E9d8BAf0edAc6f37da032`.
 
-## Expected addresses
+## Expected addresses with [Safe Singleton Factory](https://github.com/safe-global/safe-singleton-factory)
 
-- `SafeModuleSetup` at `0x2dd68b007B46fBe91B9A7c3EDa5A7a1063cB5b47`
-- `Safe4337Module` at `0x75cf11467937ce3F2f357CE24ffc3DBF8fD5c226`
+- `SafeModuleSetup` at `0xF2C2C95C168Ef121b052B4c518889CFe166cB4Fa`
+- `Safe4337Module` at `0x68C77b778228823a06e3CbcDA816d46D80B1C728`
 
 ## Changes
```

Comment:
Since the version is unreleased, everything above this section is not really worth documenting (as it will likely change before the release is done).

Reply:
(see suggestion below).

Reply:
Done 👍🏾